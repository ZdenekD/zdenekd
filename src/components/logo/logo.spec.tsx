import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Logo from '.';

describe('Logo', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Logo />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-logo');

        expect(component.exists()).toBe(true);
    });
});
