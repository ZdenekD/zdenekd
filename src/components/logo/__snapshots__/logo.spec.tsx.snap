// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo match snapshot 1`] = `
<div class="wrapper ">
  <svg data-filename="src/assets/images/logo.svg">
  </svg>
  <i class="shadow"
     style="opacity:1"
  >
  </i>
</div>
`;
